src/sim-host/protocol/socket.js

```diff
@@ -7,6 +7,10 @@ var socket;
 module.exports.initialize = function (pluginHandlers, serviceToPluginMap) {
     socket = io();
     module.exports.socket = socket;
+    module.exports.notifyPluginsReady = function () {
+        telemetry.registerPluginServices(serviceToPluginMap);
+        socket.emit('register-simulation-host');
+    }
```

Comment:
You can ignore this message :) I guess that it can be easy to read and get a description of the public/exported API of a module, if that is defined at the end, or at module level. This module is not so big, but if it grows, then It could be harder to find what are the functions that this module exports. I've seen in other modules of this project, declarations like the following:
```
Object.defineProperty(module.exports, 'socket', {
    get: function () {
        return socket; // it can be undefined if it is called before initialize
    }
});
module.exports.initialize = function () {
    // ... 
};
module.exports.notifyPluginsReady = function () {
    // same as you define but maybe to avoid calling this function before initailize, validate for socket instance
};
```
Maybe with this type you can achieve the same result.